src/views/PostPage.vue

```diff
@@ -0,0 +1,29 @@
+<template>
+  <div class="page">
+    <h1 class="entry-title">ARTICLES</h1>
+    <div class="content" v-html="content">
```

Comment:
This div should probably be closed before the Article feed. So it will be:
`<div class="content" v-html="content"></div>
<Articles />`